frontend/src/boot/i18n.ts

```diff
@@ -0,0 +1,24 @@
+import { boot } from 'quasar/wrappers';
+import messages from 'src/i18n';
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+import Quasar from 'quasar'
+
+Vue.use(VueI18n);
+console.log(Quasar.lang.getLocale())
```

Comment:
Let's use our `window.logger` class instead
```suggestion
window.logger.info(Quasar.lang.getLocale())
```

You can also use `.debug` or `.warn`, but `.info` feels right

Reply:
L24 of `i18n.ts` still uses a console.log instead of `window.logger`

Reply:
Fixed with https://github.com/ScopeLift/umbra-protocol/pull/331/commits/9f61a102ac27aa004633a5ffa4abd2a6b054c772